corehq/apps/hqadmin/views/users.py

```diff
@@ -92,21 +90,23 @@ def post(self, request, *args, **kwargs):
             users = form.cleaned_data['users']
             is_superuser = 'is_superuser' in form.cleaned_data['privileges']
             is_staff = 'is_staff' in form.cleaned_data['privileges']
-            changed_field_logs = []
+            field_changed = {}
```

Comment:
Nitpick: `fields_changed` so it matches the kwarg

Reply:
https://github.com/dimagi/commcare-hq/pull/29890/commits/623879ee01d05911bf1d5059524e262432ccdb05